python/cac_tripplanner/cac_tripplanner/views.py

```diff
@@ -1,10 +1,9 @@
 from django.shortcuts import render_to_response
+from django.template import RequestContext
+from django.conf import settings
 
 def home(request):
-    return render_to_response('home.html')
+    return render_to_response('home.html', dict(debug=settings.DEBUG))
```

Comment:
Yea, we did too -- couldn't quite get it to work though. The docs seemed to suggest you specifically had to pass in `RequestContext(request)`, but that didn't seem to work either.